### PR TITLE
update linux run-as permissions

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ There are two run modes:
 
 ### `run` Mode
 
-`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with a command-line option. The OS is automatically configured to treat the nameserver as primary. You may inspect the resulting configuration with these commands.
+`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with command-line option `--dns-upstream`. `systemd-resolved`, if enabled, is automatically configured to treat the Ziti nameserver as primary. You may inspect the configuration with these commands.
```
Comment (scareything) [inline docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md]: when running under systemd-resolved, we associate our name server with the tun interface. I wouldn't call this "primary" though. That sounds more like we are the system's global name server, similar to resolv.conf

File: docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ If the DNS record exists it returns the answer and sets query status to `NO_ERRO
 
 #### System Requirements for Mode `run`
 
-`ziti-edge-tunnel run` requires elevated privileges for managing the `/dev/net/tun` device, running `resolvectl` to assign nameservers and domain routes to the tun interface, and running `ip route` to manage IP routes. This requires running as root because `setcaps` are not inherited in all of these cases, even when the inherit bit is set.
+`ziti-edge-tunnel run` requires elevated privileges for managing the `/dev/net/tun` device, running `resolvectl` to assign nameservers and domain routes to the tun interface, and running `ip route` to manage IP routes. This is normally accomplished by PolKit rules defined in the DEB and RPM packages that set capabilities inherited by the run-as user, `ziti`.
```
Comment (scareything) [inline docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md]: there are actually two kinds of permissions that are required. Each is configured differently:

- tun and route management are permitted by passing the `CAP_NET_ADMIN` to the ziti-edge-tunnel process via `AmbientCapabilities` in the systemd service unit.
- systemd-resolved DNS configuration is permitted by PolKit.

**Comment by qrkourier** [comment]
image

**Review by dovholuknf — APPROVED** [review]
I really do prefer OpenZiti to Ziti

**Comment by qrkourier** [comment]
> I really do prefer OpenZiti to Ziti

I'll make that change in https://github.com/openziti/ziti-doc/pull/613

**Pull request merged**